lib/dbservice/schools/enrollment_record.ex

```diff
@@ -7,19 +7,18 @@ defmodule Dbservice.Schools.EnrollmentRecord do
 
   alias Dbservice.Users.Student
   alias Dbservice.Schools.School
-  alias Dbservice.Groups.Group
 
   schema "enrollment_record" do
     field(:academic_year, :string)
     field(:grade, :string)
     field(:is_current, :boolean, default: false)
     field(:board_medium, :string)
-    field(:date_of_school_enrollment, :date)
-    field(:date_of_group_enrollment, :date)
+    field(:date_of_enrollment, :date)
+    field(:group_id, :integer)
+    field(:group_type, :string)
```

Comment:
What is the need of `group_type` here?

Reply:
For easier querying. We won't have to join the `group_type` table.